zaza/openstack/charm_tests/cinder/tests.py

```diff
@@ -30,15 +33,15 @@
 )
 
 
-class CinderTests(test_utils.OpenStackBaseTest):
-    """Encapsulate Cinder tests."""
+class CinderBase(test_utils.OpenStackBaseTest):
+    """Base for Cinder tests."""
 
     RESOURCE_PREFIX = 'zaza-cindertests'
 
     @classmethod
     def setUpClass(cls):
         """Run class setup for running tests."""
-        super(CinderTests, cls).setUpClass(application_name='cinder')
+        super(CinderBase, cls).setUpClass(application_name='cinder')
```

Comment:
Just out of interest, as we are Py3 only in this library we can use the shortcut version of this:

```python
super().setUpClass(application_name='cinder')
```